konnectd/ui/src/app.css

```diff
@@ -0,0 +1,104 @@
+/* additional css on top of kpop */
+html {
+  font-family: 'Open Sans', sans-serif;
+}
+
+strong {
+  font-weight: 600;
+}
+
+.oc-login-bg {
+  background-image: url(./images/background.jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  background-position: center;
+  z-index: 0;
+}
+
+#loader {
+  /* NOTE(longsleep): White here needed because of the background image */
+  color: white;
+  text-shadow: #000 0 0 1px;
+}
+
+.oc-logo {
+  position: absolute;
+  top: -130px;
+  left: 50%;
+  height: 80px;
+  transform: translateX(-50%);
+}
+
+.oc-progress {
+  /* Needs to be important to overwrite material-ui */
+  background-color: rgba(78,133,200,.8) !important;
```

Comment:
Codacy found an issue: [Expected single space after "," in a single-line function (function-comma-space-after)](https://app.codacy.com/gh/owncloud/ocis/pullRequest?prid=6337760)